Contrib/JaxConfGen/demo.py

```diff
@@ -0,0 +1,140 @@
+# Author: Yutong Zhao
```

Comment:
I would put a license on each file.  Perhaps BSD, it's fine to assign copyright to yourself.